R/github.R

```diff
@@ -61,7 +61,7 @@ use_github <- function(organisation = NULL,
   visibility <- match.arg(visibility)
   check_protocol(protocol)
   check_uses_git()
-  default_branch <- git_default_branch()
+  default_branch <- guess_local_default_branch()
```

Comment:
I think this a reasonable substitution because I can't see why origin or upstream would be setup _before_ you're calling `use_github()`. Please let me know if I've forgotten some likely scenario.